File: HackerRank/Python/Easy/E0021.py
# Problem Statement: https://www.hackerrank.com/challenges/python-string-formatting/problem

def print_formatted(N):
    l = N.bit_length()
    for i in range(1, N + 1):
        print(f'{i:{l}d} {i:{l}o} {i:{l}X} {i:{l}b}')

File: HackerRank/C++/Easy/E0021.cpp
/*
Problem Statement: https://www.hackerrank.com/challenges/inheritance-introduction/problem
*/

#include <iostream>

using namespace std;

class Triangle {
public:
	void triangle() {
		cout << "I am a triangle" << endl;
	}
};

class Isosceles : public Triangle {
public:
	void isosceles() {
		cout << "I am an isosceles triangle" << endl;
	}
	
	void description() {
		cout << "In an isosceles triangle two sides are equal" << endl;
	}
};

int main() {
	Isosceles isc;
	isc.isosceles();
  	isc.description();
	isc.triangle();
	return 0;
}